From e324855fdcff45ddaea246d4d1d2f0e14b98df26 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Thu, 31 Jan 2019 19:16:18 +0900
Subject: [PATCH 4/4] docs: add usage of server add command in usecase

In `Multiple Nodes` section in usecase, add an example of adding nodes
by using `server add` command as usage.

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 docs/guides/use_cases/multi_nodes.rst | 20 ++++++++++++++++++--
 1 file changed, 18 insertions(+), 2 deletions(-)

diff --git a/docs/guides/use_cases/multi_nodes.rst b/docs/guides/use_cases/multi_nodes.rst
index 6be567e..b44d0bd 100644
--- a/docs/guides/use_cases/multi_nodes.rst
+++ b/docs/guides/use_cases/multi_nodes.rst
@@ -49,8 +49,24 @@ for each of hosts. SPP CLI is able to be launched on any of nodes.
         -b 192.168.11.103 \
         -b 192.168.11.104 \
 
-If you succeeded to launch all of processes before, you can find them
-by running ``sever list`` command.
+Or you can add nodes after launching SPP CLI. Here is an example of
+launching it with first node, and adding the rest of nodes after.
+
+.. code-block:: console
+
+    # Launch SPP CLI
+    $ python src/spp.py -b 192.168.11.101
+    Welcome to the spp.  Type help or ? to list commands.
+
+    spp > server add 192.168.11.102
+    Registered spp-ctl "192.168.11.102:7777".
+    spp > server add 192.168.11.103
+    Registered spp-ctl "192.168.11.103:7777".
+    spp > server add 192.168.11.104
+    Registered spp-ctl "192.168.11.104:7777".
+
+If you have succeeded to launch all of ``spp-ctl`` processes before,
+you can find them by running ``sever list`` command.
 
 .. code-block:: console
 
-- 
2.7.4

